src/Database/Schema/CollectionInterface.php

```diff
@@ -21,12 +21,18 @@
  *
  * Used to access information about the tables,
  * and other data in a database.
+ *
+ * @method array<string> listTablesAndViews() Get the list of tables available in the current connection.
+ * This will include any views in the schema.
+ * @method array<string> listTablesWithoutViews() Get the list of tables available in the current connection.
+ * This will exclude any views in the schema.
```

Comment:
If this is expected to exist only in Schema\Collection, not sure we should add an annotation on the interface. Does anything else implement CollectionInterface?